book/src/drivers.md

```diff
@@ -95,6 +95,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Oracle Cloud Infrastructure(OCI) Block Storage](https://github.com/oracle/oci-cloud-controller-manager/blob/master/container-storage-interface.md) | `blockvolume.csi.oraclecloud.com` | v1.1 | A Container Storage Interface (CSI) Driver for Oracle Cloud Infrastructure (OCI) Block Storage | Persistent | Read/Write Single Pod | Yes | Topology
 [oVirt](https://github.com/openshift/ovirt-csi-driver) | `csi.ovirt.org` | v1.0 | A Container Storage Interface (CSI) Driver for [oVirt](https://ovirt.org)  | Persistent | Read/Write Single Pod | Yes | Block, File Storage
 [Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | `pxd.portworx.com` | v1.4 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent and Ephemeral | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Raw Block, Cloning
+[Proxmox](https://github.com/sergelogvinov/proxmox-csi-plugin) | `csi.proxmox.sinextra.dev` | v1.6 | A Container Storage Interface (CSI) Driver for [Proxmox](https://www.proxmox.com/en/) | Persistent | Read/Write Single Pod | Yes | Expansion, Topology
```

Comment:
I see here: https://github.com/sergelogvinov/proxmox-csi-plugin/blob/main/go.mod#L7
github.com/container-storage-interface/spec v1.9.0

That means column 3 should be v1.9 instead of v1.6.